src/parsec_client/operations/psa_asymmetric_decrypt.md

```diff
@@ -27,11 +27,17 @@ Decrypt a short message with a private key. Opcode: 11 (`0x000B`)
 
 - `PsaErrorNotPermitted`: The key does not have the `decrypt` flag, or it does not permit the
    requested algorithm.
+- `PsaErrorInvalidPadding`: The decrypted padding is incorrect. See Warning below.
 
 ## Description
 
 This function will decrypt a short message with the private key of the provided key pair.
 
+**WARNING:** In some protocols, when decrypting data, it is essential that the behavior of the
```

Comment:
Nit: not sure about the standard (American vs. British English) we use in Parsec book.
The rest of the parsec book use the spelling `behaviour`.

Reply:
For Arm as an organization the standard is US English but the Parsec book seems to contain a mix e.g. [virtualisation](https://parallaxsecond.github.io/parsec-book/archive/system_architecture.html?highlight=virtualisation#introduction) versus [virtualization](https://parallaxsecond.github.io/parsec-book/overview.html?highlight=virtualization#why-platform-agnostic-security). Happy to defer until next release if issue raised (low priority for me)